site/scripts/copy-blocks-to-ci-cache.ts

```diff
@@ -0,0 +1,37 @@
+import chalk from "chalk";
```

Comment:
This command is pretty much a copy-pasted version of `site/scripts/copy-blocks-from-ci-cache.ts`. This is not very DRY, but I don’t think it’s worth introducing abstractions to deduplicate this code. It is temporary (we won’t store block cache in Vercel forever).